### PR TITLE
[Samples][nlp-with-dispatch] Create main structure

### PR DESCRIPTION
**Pull request by fran893**
Fixes #

## Specific Changes

  - 
  -
  -

### CODE REVIEW

File: samples/14.nlp-with-dispatch/pom.xml [hunk 1]
```diff
@@ -0,0 +1,248 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.microsoft.bot.sample</groupId>
+    <artifactId>bot-nlpwithdispatch</artifactId>
```
[the remark this answers is not in the document]
Comment (fran893) [reply samples/14.nlp-with-dispatch/pom.xml:9]: I followed this approach reviewing other samples like these ones:

- [usingcards](https://github.com/microsoft/botbuilder-java/tree/main/samples/06.using-cards/src/main/java/com/microsoft/bot/sample/usingcards)
- [multiturnprompt](https://github.com/microsoft/botbuilder-java/tree/main/samples/05.multi-turn-prompt/src/main/java/com/microsoft/bot/sample/multiturnprompt)

But I can what you said.

File: samples/14.nlp-with-dispatch/src/main/java/com/microsoft/bot/sample/nlp/with/dispatch/Application.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.microsoft.bot.sample.nlpwithdispatch;
```
Comment (Batta32) [inline samples/14.nlp-with-dispatch/src/main/java/com/microsoft/bot/sample/nlp/with/dispatch/Application.java:1]: This should be changed, right?

File: samples/14.nlp-with-dispatch/src/test/java/com/microsoft/bot/sample/nlp/with/dispatch/ApplicationTest.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package com.microsoft.bot.sample.nlpwithdispatch;
```
Comment (Batta32) [inline samples/14.nlp-with-dispatch/src/test/java/com/microsoft/bot/sample/nlp/with/dispatch/ApplicationTest.java:1]: This should be changed, right?

**Review by Batta32 — CHANGES_REQUESTED** [review]

File: samples/14.nlp-with-dispatch/src/main/java/com/microsoft/bot/sample/nlp/with/dispatch/Application.java (hunk 2 above)
Comment (fran893) [reply samples/14.nlp-with-dispatch/src/main/java/com/microsoft/bot/sample/nlp/with/dispatch/Application.java:1]: Yes, I forgot that

**Review by Batta32 — APPROVED** [review]

**Pull request merged**